### PR TITLE
Add issues template for bugs and feature requests

### PR DESCRIPTION
**Pull request by imatiach-msft**
Add issues templates for bugs and features for users.  I just used the default templates provided by github.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+name: Bug report
+about: Create a report to help us improve
+title: ""
+labels: ""
+assignees: ""
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
```
Comment (gaugup) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]: should we add a section for SDK exception stack trace?

**Review by tongyu-microsoft — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (imatiach-msft) [reply .github/ISSUE_TEMPLATE/bug_report.md:11]: done!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+
+- OS: [e.g. iOS]
+- Browser [e.g. chrome, safari]
+- Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (romanlutz) [inline .github/ISSUE_TEMPLATE/bug_report.md:35]: I think it's fairly safe to say nobody uses our dashboard on the phone? 🙂

Comment (imatiach-msft) [reply .github/ISSUE_TEMPLATE/bug_report.md:35]: removed phone section

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+**Stack trace**
+If applicable, please add the sdk stack trace for the error.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
[the remark this answers is not in the document]
Comment (imatiach-msft) [reply .github/ISSUE_TEMPLATE/bug_report.md:29]: done!

**Review by romanlutz — APPROVED** [review]

**Pull request merged**